modules/tinymce/src/core/main/ts/api/commands/HistoryCommands.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import type Editor from '../Editor';
+
+export const registerCommands = (editor: Editor) => {
+  editor.editorCommands.addCommands({
+    mceAddUndoLevel: () => {
+      editor.undoManager.add();
```

Comment:
Should we extract `editor.undoManager` out to a separate variable here? It'd minify better than, though granted it likely wouldn't make much difference if we include GZIP compression.

Reply:
We can't since the editor.undoManager gets initialized after the commands gets registered. I don't want to change that order now.